src/servers/grpc_server.cc

```diff
@@ -1825,15 +1809,15 @@ GRPCServer::Start()
       "InferHandler", server_, server_id_, trace_manager_, smb_manager_,
       &service_, infer_cq_.get(),
       infer_allocation_pool_size_ /* max_state_bucket_count */);
-  hinfer->Start(infer_thread_cnt_);
+  hinfer->Start(/* infer_thread_cnt_ FIXME */ 1);
```

Comment:
It looks like you missed reverting these.

Reply:
Yes, I need to file a separate ticket for these...